### ISSUE TITLE
Implementing nested donut chart

### ISSUE THREAD

**Issue by madhy** [novus/nvd3#1840]
Hi,
There is a requirement to show  donut chart in nested form (one inside another). Please let me provide hints on how to implement nested donut chart.

Thanks in advance.
Madhy

**Comment by liquidpele** [comment]
do you mean like the sunburst chart? 

https://nvd3-community.github.io/nvd3/examples/sunburst.html

**Comment by madhy** [comment]
Yes. Need to show similar like the attached image. I need to use NVD3 and to  show two nested rings. It would be great if i get the data set structure from anyone at the earliest.

Thanks in advance.

![download](https://cloud.githubusercontent.com/assets/8539097/19241245/43fdd53c-8f2b-11e6-841c-cc4b36448b53.jpg)
